### DISCUSSION TITLE
Add a "Skip Feedback" option

### DISCUSSION THREAD

**Discussion by DaRkViVi** [ChristopherKlay/StadiaEnhanced#76] [Ideas & Requests]
As you know, each time (or at least most of them) you end a game on Stadia you get the quality feedback.
Ok, it's just a click, but we are not in Beta anymore and we should be able to give it on purpose and by asking, not prompted each time.

Would it be possible to add an option to skip it automatically?

**Comment by ChristopherKlay** [comment]
Should be, looking into it (:
